src/steamship/agents/react/react.py

```diff
@@ -54,6 +54,9 @@ class ReACTAgent(LLMAgent):
 Previous conversation history:
 {chat_history}
 
+Other relevant previous conversation:
+{search_history}
```

Comment:
```suggestion
{relevant_history}
```

Reply:
Not sure if this is better. But search implies limits, not sure we want that.

Reply:
👍